docs/fieldnorm/verification_validation.md

```diff
@@ -84,3 +80,7 @@ For Human Genetics, specific GIAB samples are used for performance evaluation. I
 ## Choice of tools and/or settings
 
 A subject missing in _software verification_ and _software validation_ is how specific tools and/or settings have been selected. One could add a section to the _software validation_ document with an introduction into the issue, the choices made, the results, and the conclusions.
+
+## Builds, databases, reference sets etc. 
+
+Bioinformatic workflows often use (external) databases, reference sets, or specific genome builds. The link to them could be hardcoded, hence, updating them should follow software verification and validation as described above. The clinical validation of new database/reference set/genome build could be done either with the medical laboratories validation procedures or the software validation. Choose the kind of validation that works best for this particular update.  
```

Comment:
If I am correct, this only refers to updating the links to the databases right? I am wondering; if you link to an (external), should you then also monitor these databases themselves? e.g. I am now working with a program that links to external datasets, of which some have become inaccessible over time. Is there any way to monitor if the external sources are still up and good after the code has been released? Or is this outside the scope of V&V?

And if you do find out that your external database has been updated, should you be required to do another verification on your part? I have seen datasets that were brutally updated with a different format, causing some of our pipelines to behave differently.

Reply:
That could fall under "SOUPs". I think there's a risk to linking to a database that you cannot control. Think about the risks and how you mitigate them.   If it's important, than these exerternal databases could be part of what you validate with every release. (like a single test to see whether it's still there). 

It's all in the risks ;)